prognoses_monitoring_reports_code/automatic_checks/automatic_sanity_check.py

```diff
@@ -0,0 +1,84 @@
+import pandas as pd
+import numpy as np
+
+
+class CheckReport:
+    def __init__(self, ean: int):
+        self.ean = ean
+        self.something_wrong = False
+        self.reasons = []
+
+    def add_check_failed(self, reason: str) -> None:
+        self.something_wrong = True
+        self.reasons.append(reason)
+
+
+def automatic_check_forecast_da(df_month, ean):
+    """Initialises automatic check
+
+    Args:
+        df_month: pd.Dataframe with timeseries data for specific ean
+        ean: EAN of connection point.
+
+    Returns: Checkreport object that indicates if any checks failed and for what reason.
+
+    """
+    time_combined = df_month[df_month["Connection point EAN"] == ean][
+        ["Realised [MW]", "Prognosis [MW] (DA)", "Prognosis [MW] (ID)"]
+    ]
+
+    return check_if_predicted_is_off(
+        time_combined["Realised [MW]"], time_combined["Prognosis [MW] (DA)"], ean
+    )
```

Comment:
No, slicing here is better because the parent function is already quite crowded. The Ean is still needed for the report so the reports can be traced back to a specific connection point